site2/docs/functions-develop.md

```diff
@@ -129,6 +121,12 @@ public interface SerDe<T> {
     byte[] serialize(T input);
 }
 ```
+SerDe works in the following way in Java Functions.
+- If the input and output topics have schema, Pulsar Functions use schema for SerDe.
+- If the input and output topics do not exist, Pulsar Functions adopt the following rules to determine SerDe:
+  - If the schema type is specified, Pulsar Functions use the specified schema type.
+  - If SerDe is specified, Pulsar Functions use the specified SerDe, and the schema type for input and output topics is `Byte`.
+  - If either the schema type nor SerDe is specified, Pulsar Functions use the built-in SerDe. For non-primitive schema type, the built-in SerDe serializes and deserializes objects in the `json` format. 
```

Comment:
either...or, neither...nor, Please double confirm which one you should use.

Reply:
neither ... nor...
Nice catch, thank you.